diff --git a/agent/config/config.go b/agent/config/config.go
index ccbafa793..83496304f 100644
--- a/agent/config/config.go
+++ b/agent/config/config.go
@@ -133,7 +133,7 @@ const (
 	DefaultTaskMetadataBurstRate = 60
 
 	//Known cached image names
-	CachedImageNameAgentContainer = "amazon/amazon-ecs-agent:latest"
+	CachedImageNameAgentContainer = "jazzl0ver/amazon-ecs-agent:latest"
 
 	// DefaultNvidiaRuntime is the name of the runtime to pass Nvidia GPUs to containers
 	DefaultNvidiaRuntime = "nvidia"
